sphinx_gallery/tests/test_gen_rst.py

```diff
@@ -1239,6 +1239,5 @@ def test_newlines(log_collector_wrap):
     assert tee.newlines == tee.output.newlines
 
 
-# TODO: test that broken thumbnail does appear when needed
```

Comment:
Since the new test also looks at the default behaviour for expected failing examples, can remove this `TODO` from elsewhere.